src/CommAggregation.chpl

```diff
@@ -388,4 +396,225 @@ module CommAggregation {
     const cachePaddedLocales = (numLocales + 7) & ~7;
     return c_aligned_alloc(int, 64, cachePaddedLocales);
   }
+
+  module BigIntegerAggregation {
```

Comment:
Is there a reason that this is going to its own module?

Reply:
Easier to disable if we need to and makes it easier to eventually upstream.